### ISSUE TITLE
Messages not showing on application chat for new users

### ISSUE THREAD

**Issue by brnsolikyl** [yunity/karrot-frontend#1097]
I've tested the new application feature to join a group on a web browser of a laptop. On full screen it works fine, but with a smaller window the messages do not appear unless I click on "Initial questions and answers" to minimize it. Another user, Hela, in the Warsaw group experienced the same problem.

Here's a screenshot:
![image](https://user-images.githubusercontent.com/25367304/45307491-70730580-b51f-11e8-9aeb-5f3c21f2c908.png)

**Comment by nicksellen** [comment]
Hmm, I tried this on Chrome and Firefox on Linux and MacOS on dev.karrot.world and karrot.world but couldn't recreate it :/

Any tips @brnsolikyl for how to recreate it?

**Comment by brnsolikyl** [comment]
I tried on Firefox on Linux too, on karrot.world. Have you tried resizing the browser window, @nicksellen ? Because on full-sized window on my laptop it worked well, I could see the messages, but not on a smaller window. Perhaps it is also related to the size of the application text, under "Initial questions and answers". I noticed there too it is not possible to scroll down to see all the replies, only if you click to minimize the "Initial questions and answers" frame.

Is there any group on dev.karrot.world we could try it by doing an application?
